mongorm/Document.py

```diff
@@ -30,6 +30,12 @@ def save( self, forceInsert=False, **kwargs ):
 		
 		if '_id' in self._data and self._data['_id'] is None:
 			del self._data['_id']
+
+		# safe not supported in pymongo 3.0+, use w for write concern instead
+		if 'safe' in kwargs:
+			kwargs['w'] = 1 if kwargs['safe'] else 0
+			del kwargs['safe']
```

Comment:
It would be good if we could raise some kind of warning here so that we know if the engine code is using this deprecated arg

Reply:
I can add in a deprecation warning.